@pauliescanlon/gatsby-mdx-embed/src/components/Twitch/Twitch.tsx

```diff
@@ -1,49 +1,45 @@
 import React, { FunctionComponent } from 'react'
 import { getPadding } from '../../utils'
 
-export interface ITwitchProps {
+type Props = {
   /** Twitch id */
   twitchId: string
   /** Skip to a time in the video */
-  skipTo: {
+  skipTo?: {
```

Comment:
This needs to remain as an optional prop i think?

Reply:
Yeah I've made it optional now since you have a default-value